gql/lib/src/language/printer.dart

```diff
@@ -583,13 +570,13 @@ class _PrintVisitor extends Visitor<String> {
   @override
   String visitDirectiveDefinitionNode(DirectiveDefinitionNode node) => [
         if (node.description != null) ...[
-          node.description.accept(this),
+          node.description!.accept(this),
```

Comment:
I do not know NNBD that well, but this should be inferred by the if.

Reply:
It can't, probably because getting `node.description` may somehow alter its next invocation.

dereferenced the attribute to avoid `!`